dprint.json

```diff
@@ -0,0 +1,16 @@
+{
+  "excludes": [
+    // These files are auto-generated and don't need to be formatted
+    "./types/vscode"
+  ],
+  "plugins": [
+    "https://plugins.dprint.dev/typescript-0.85.1.wasm"
+  ],
+  "typescript": {
+    "arrowFunction.useParentheses": "preferNone",
+    "indentWidth": 4,
+    "lineWidth": 120,
+    "quoteStyle": "alwaysSingle",
```

Comment:
\[a thread for replies to [#66238 (comment)](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66238#issuecomment-1659372438)]

Reply:
> 👋 @JoshuaKGoldberg! TypeScript repo itself uses double quotes:
[microsoft/TypeScript](https://github.com/microsoft/TypeScript)
>
> Prettier defaults to double quotes too (I know it’s not used, just giving some context).
>
> Given the above and the number of conversions from " to ', WDYT of using double quotes instead?
>
> If you are interested in seeing opinions on pros and cons of each choice, here is a rather comprehensive discussion thread: https://github.com/prettier/prettier/issues/4102

Thanks for the context! I've been going off the _Formatting Configuration_ section in https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/65993. This was the closest equivalent to the existing https://github.com/DefinitelyTyped/DefinitelyTyped/blob/59041d42631e3d6691cfb961b1d161c4efe61c41/.prettierrc.json. That file is there as an editor hint and a lot of contributors' Prettier extensions have used it as a result.

I posted a comparison back in the discussion: https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/65993#discussioncomment-6620497

Reply:
Following up: nobody felt strongly one way or the other, and a few folks were in favor of removing dprint's overrides (i.e. going with the default). So I switched to removing the override.